### PR TITLE
New package: CLBlast

### PR DESCRIPTION
**Pull request by peterbud**
Two packages:
* **OpenCL-ICD**: this is an OpenCL ICD Loader, it compiles and installs an OpenCL shared and static library, subsequent OpenCL based libraries can link to the static version (this is also useful for Appveyor CI, which does not have OpenCL ICD at all).
* **CLBlast** package itself

### CODE REVIEW

File: mingw-w64-opencl-icd-git/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+provides=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+replaces=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=15.261c128
+pkgrel=1
+pkgdesc="OpenCL ICD Loader (mingw-w64)"
+arch=('any')
+url='https://github.com/KhronosGroup/OpenCL-ICD-Loader'
+license=('BSD')
+depends=("${MINGW_PACKAGE_PREFIX}-opencl-headers")
+makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
```
Comment (Alexpux) [inline mingw-w64-opencl-icd-git/PKGBUILD]: Add git to makedepends

File: mingw-w64-opencl-icd-git/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Maintainer: Peter Budai <peterbudai at hotmail.com>
+
+_realname=opencl-icd
+pkgbase=mingw-w64-${_realname}-git
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-git"
+provides=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+replaces=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=15.261c128
```
Comment (Alexpux) [inline mingw-w64-opencl-icd-git/PKGBUILD]: You need write ````pkgver```` function, look to other git based packages

**Comment by peterbud** [comment]
@Alexpux is there any problem with this PR? Just let me know if you feel we don't need this package, and then I close this PR.

File: mingw-w64-opencl-icd-git/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# Maintainer: Peter Budai <peterbudai at hotmail.com>
+
+_realname=opencl-icd
+pkgbase=mingw-w64-${_realname}-git
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-git"
+provides=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+replaces=("${MINGW_PACKAGE_PREFIX}-${_realname}")
```
Comment (Alexpux) [inline mingw-w64-opencl-icd-git/PKGBUILD]: Remove this line. This Im adding to release packages when switch from git

Comment (peterbud) [reply mingw-w64-opencl-icd-git/PKGBUILD]: I have removed the "replaces" line

**Pull request merged**